src/dso_api/dynamic_api/serializers.py

```diff
@@ -399,6 +402,7 @@ def _get_hal_field_names(self, fields):  # noqa: C901
         ]
         for field_name, field in fields.items():
             if isinstance(field, TemporalHyperlinkedRelatedField):
+
```

Comment:
beetje vreemd, waarom die newlines?

Reply:
Is nog een leftover van debugging denk ik. Verwijderd.